bin/faucet/frontend/index.js

```diff
@@ -236,9 +238,9 @@ class MidenFaucet {
             completedPublicModal.classList.add('active');
 
             const explorerButton = document.getElementById('explorer-button');
-            if (mintingData.explorer_url) {
+            if (this.explorer_url) {
                 explorerButton.style.display = 'block';
-                explorerButton.onclick = () => window.open(mintingData.explorer_url + '/tx/' + mintingData.tx_id, '_blank');
+                explorerButton.onclick = () => window.open(this.explorer_url + '/tx/' + mintingData.tx_id, '_blank');
```

Comment:
Does this get correctly concatenated? The default URL has a trailing `/` and then you are concatenating against a `/tx` (although I don't know how a `Url` gets serialized).